@here/harp-mapview/lib/geometry/TileGeometryLoader.ts

```diff
@@ -187,13 +182,13 @@ export class TileGeometryLoader {
      * Start with or continue with loading geometry. Called repeatedly until `isFinished` is `true`.
      */
     update(
-        enabledKinds: GeometryKindSet | undefined,
-        disabledKinds: GeometryKindSet | undefined
+        enabledKinds?: GeometryKindSet | undefined,
+        disabledKinds?: GeometryKindSet | undefined
```

Comment:
i guess now the union type with undefined is not needed anymore?

Reply:
done